cluster/manifests/kube-cluster-autoscaler/autoscaler-deployment.yaml

```diff
@@ -28,7 +28,7 @@ spec:
         effect: NoSchedule
       containers:
       - name: cluster-autoscaler
-        image: registry.opensource.zalan.do/teapot/kube-cluster-autoscaler:v1.1.1
+        image: registry.opensource.zalan.do/teapot/kube-cluster-autoscaler:v1.1.2
```

Comment:
I don't see this image in our registry. For some reason the e2e didn't fail because of this...